### PR TITLE
jannate.rahim

### PR DESCRIPTION
**Pull request by jannate-r**

### CODE REVIEW

File: src/main/java/view/HomeView.java
Comment (Kurochiisu) [inline src/main/java/view/HomeView.java:1]: listeners and actions can be combined. I think you can also introduce a single method for updating the labels to improve readability.

**Review by Kurochiisu — DISMISSED** [review]

File: src/main/java/use_case/display_home/DisplayHomeInputData.java
[the remark this answers is not in the document]
Comment (daniilio) [reply src/main/java/use_case/display_home/DisplayHomeInputData.java:1]: Yep, this still needs to be fixed Jannate. Please push your changes when you are done.

**Review by daniilio — CHANGES_REQUESTED** [review]
Please fix remove the boolean variables for the buttons in DisplayHomeInputData as Chika requested. Please also remove any mention of WeatherDataFactory from the DisplayHomeInteractor. It shouldn't be using a WeatherDataFactory.

**Review by daniilio — APPROVED** [review]
Looks good.

**Pull request merged**